src/main/java/thaumicenergistics/common/storage/EnumEssentiaStorageTypes.java

```diff
@@ -22,19 +22,19 @@ public enum EnumEssentiaStorageTypes {
             ThEStrings.Item_StorageComponent_16k),
     Type_64K(3, 1 << 16, 12, "64k", EnumRarity.rare, 2.0, ThEStrings.Item_EssentiaCell_64k,
             ThEStrings.Item_StorageComponent_64k),
-    Type_256K(4, 1 << 18, 24, "256k", EnumRarity.rare, 2.5, ThEStrings.Item_EssentiaCell_256k,
+    Type_Creative(4, 0, 63, "creative", EnumRarity.epic, 0.0, ThEStrings.Item_EssentiaCell_Creative, null),
```

Comment:
I know it's 63 base, but can we move this up? If it's to stay as a base 2, maybe 127? Otherwise 95 or something would work, just to future proof the essentia creative cells

Reply:
I remember that somewhere limit the max size to 63, if is upper than this will force change it to 63.

Reply:
Ah that's lame as hell, carry on then lmfao

Reply:
I will try to improve the limit tomorrow.

Reply:
this pr is just fine. It only have to change back the ids. if we change values we need to wait after stable